tests/test_utils_misc/__init__.py

```diff
@@ -95,14 +96,44 @@ class TestItem(Item):
             list(arg_to_iter(TestItem(name="john"))), [TestItem(name="john")]
         )
 
-    def test_create_instance(self):
```

Comment:
I also think the create_instance test should be kept as long as we keep the function itself.